suite-common/suite-config/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from './features';
```

Comment:
Can you also move `packages/suite/src/config/suite/fiat.ts` here? @juriczech  will need it soon so let's do it now